### ISSUE TITLE
Life on Mars saves not working

### ISSUE THREAD

**Issue by bootsector** [MiSTer-devel/Genesis_MiSTer#225]
Probably because MiSTer lacks support for EPROM saves?

**Comment by birdybro** [comment]
PCB:

![image](https://user-images.githubusercontent.com/16388068/215393603-449084aa-6293-4d03-9535-a0d26fddceee.png)

Flash chip:

https://www.infineon.com/dgdl/Infineon-S29GL064N_S29GL032N_64_Mbit_32_Mbit_3_V_Page_Mode_MirrorBit_Flash-DataSheet-v03_00-EN.pdf?fileId=8ac78c8c7d0d8da4017d0ed556fd548b

It looks like it's using this chip for the ROM and saves combined?

This would be a lot different than EEPROM

**Comment by bootsector** [comment]
I'm pretty sure the latest ROM produces a .sav file when saving, so I wouldn't say it's combining saves along with the ROM.

**Comment by birdybro** [comment]
Right. Hopefully someone with a modern version of this can provide PCB pictures, since the ones from RetroRGB don't show any sram/eeprom/fram and don't show a backup battery.
